### PR TITLE
Added variance calculation for all four quads in pulse calibration

### PR DESCRIPTION
**Pull request by matthewware**

### CODE REVIEW

**Comment by matthewware** [comment]
Maybe I should add a test function for all four quadratures?

**Comment by dieris** [comment]
That's right for amp/phase if we assume that the distributions are Gaussian. I think we should add that disclaimer

File: test/test_pulsecal.py [hunk 1]
```diff
@@ -206,13 +206,50 @@ def update_data(amp, ct):
         #save simulated data
         np.save(self.filename, ideal_data)
         # Verify output matches what was previously seen by matlab
-        pi_cal = cal.PiCalibration(self.q.label, numPulses)
+        pi_cal = cal.PiCalibration(self.q.label, numPulses, quad='imag')
```
Comment (dieris) [inline test/test_pulsecal.py]: why this choice as default? The kernel is typically optimized to put the signal in the real quadrature

Comment (matthewware) [reply test/test_pulsecal.py]: No real reason.  I'm wondering if it's worth the time in CI to test this for all four quadratures.  The only down side is the amount of time it takes.

Comment (matthewware) [reply test/test_pulsecal.py]: If we have to chose one it should be real but I wanted to test one of the new quadratures as well.  I'll just add all four for now.

Comment (dieris) [reply test/test_pulsecal.py]: Oh I didn't realize that this was just the test :)

**Pull request merged**